default-config.json

```diff
@@ -72,6 +72,16 @@
         "format": "Hi %u, I'm CadenceBot.",
         "continues": false,
         "replyOnly": false
+      },
+      "Cadence am I ": {
+        "random": [
+            "Yes, you absolutely are _%s_.",
+            "Yes, you are _%s_, although, Jakob is even more _%s_.",
```

Comment:
Would you like to/would you like me to change this to mention him instead of saying his name?

(also, maybe this should be in the private config file instead of on github for the world to see....)

Reply:
oh how do i make it private lol

Reply:
The server admin has to do it in config.json...

I've been looking for a good test-case for how randoms functions when overriding, so I'll use this. If you could remove it from this PR, I'll merge it and add it on the live private config.

If it doesn't get added to the options, I'll add code to `recursiveDefault` to merge arrays instead of choosing one.

Reply:
what do i need to remove lol

Reply:
I'd like you to remove the response option that mentions Jakob (Yes you are, although jakob is even more). I'll then merge it and re-add it in my private config (where it will be a testbed for how I want to be able to customize these arrays).

If we were being good, we'd rebase the line out of history entirely - Which I'll help you with if you want to do - and then use @kenellorando's favorite command, the force-push, so it doesn't appear on github entirely.

But that isn't strictly necessary,